pkg/nuctl/test/project_test.go

```diff
@@ -262,19 +249,79 @@ func (suite *projectExportImportTestSuite) TestImportProjects() {
+	}
 
 	defer suite.ExecuteNuctl([]string{"delete", "fe", function1EventName}, nil) // nolint: errcheck
 	defer suite.ExecuteNuctl([]string{"delete", "fe", function2EventName}, nil) // nolint: errcheck
 	defer suite.ExecuteNuctl([]string{"delete", "fe", function3EventName}, nil) // nolint: errcheck
 	defer suite.ExecuteNuctl([]string{"delete", "fe", function4EventName}, nil) // nolint: errcheck
 }
 
+func (suite *projectExportImportTestSuite) TestImportProject() {
+	apiGatewaysEnabled := suite.origPlatformKind == "kube"
+
+	uniqueSuffix := "-" + xid.New().String()
+	projectConfigPath := path.Join(suite.GetImportsDir(), "project.yaml")
+	projectName := "test-project" + uniqueSuffix
+	function1Name := "test-function-1"
+	function2Name := "test-function-2"
+	function1EventDisplayName := "test-function-event-1"
+	function2EventDisplayName := "test-function-event-2"
+	apiGateway1Name := "test-api-gateway-1"
+	apiGateway2Name := "test-api-gateway-2"
+
+	uniqueProjectConfigPath := suite.addUniqueSuffixToImportConfig(projectConfigPath,
+		uniqueSuffix,
+		[]string{function1Name, function2Name},
+		[]string{function1EventDisplayName, function2EventDisplayName},
+		//[]string{apiGateway1Name})
+		[]string{apiGateway1Name, apiGateway2Name})
+	defer os.Remove(uniqueProjectConfigPath) // nolint: errcheck
+
+	function1Name = function1Name + uniqueSuffix
+	function2Name = function2Name + uniqueSuffix
+	function1EventDisplayName = function1EventDisplayName + uniqueSuffix
+	function2EventDisplayName = function2EventDisplayName + uniqueSuffix
+	apiGateway1Name = apiGateway1Name + uniqueSuffix
+	apiGateway2Name = apiGateway2Name + uniqueSuffix
+
+	defer suite.ExecuteNuctl([]string{"delete", "proj", projectName}, nil) // nolint: errcheck
+	defer suite.ExecuteNuctl([]string{"delete", "fu", function1Name}, nil) // nolint: errcheck
+	defer suite.ExecuteNuctl([]string{"delete", "fu", function2Name}, nil) // nolint: errcheck
+
+	if apiGatewaysEnabled {
+		defer suite.ExecuteNuctl([]string{"delete", "agw", apiGateway1Name}, nil) // nolint: errcheck
+		defer suite.ExecuteNuctl([]string{"delete", "agw", apiGateway2Name}, nil) // nolint: errcheck
+	}
+
+	// import the project
+	err := suite.ExecuteNuctl([]string{"import", "proj", uniqueProjectConfigPath, "--verbose"}, nil)
+	suite.Require().NoError(err)
+
+	suite.assertProjectImported(projectName)
+	suite.assertFunctionImported(function1Name, true)
+	suite.assertFunctionImported(function2Name, true)
+	function1EventName := suite.assertFunctionEventExistenceByFunction(function1EventDisplayName, function1Name)
+	function2EventName := suite.assertFunctionEventExistenceByFunction(function2EventDisplayName, function2Name)
+
+	if apiGatewaysEnabled {
+		suite.verifyAPIGatewayExists(apiGateway1Name, function1Name)
+		suite.verifyAPIGatewayExists(apiGateway2Name, function2Name)
+	}
+
+	defer suite.ExecuteNuctl([]string{"delete", "fe", function1EventName}, nil) // nolint: errcheck
```

Comment:
Please add a comment why is this needed - what created these fe ?

Reply:
They were created few lines ago as part of project import.
I'll add a comment about this 👍